protocol/rpcprovider/rpcprovider_server.go

```diff
@@ -382,7 +382,8 @@ func (rpcps *RPCProviderServer) verifyRelaySession(ctx context.Context, request
 		} else if request.RelaySession.Epoch == 0 {
 			errorMessage = "user reported lava block 0, either it's test rpcprovider or a consumer that has no node access"
 		}
-		return nil, nil, utils.LavaFormatWarning(errorMessage, lavasession.EpochMismatchError,
+		return nil, nil, utils.LavaFormatInfo(errorMessage,
```

Comment:
this can;t be changed you have to return lavasession.EpochMismatchError here not a generic error, also your print of a specific constant does not provide information